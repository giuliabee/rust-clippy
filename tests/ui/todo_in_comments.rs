// todo 0

#![warn(clippy::todo_in_comments)]
// TODO: 1

// ciao bubi

/*
TODO: 2
 */
fn main() {
    // TODO: 3
    /*
    TODO: 4
     */
}

// TODO: 5

/*
TODO: multi
line
comment
containing
todo
 */

/*
 * TODO
 */
