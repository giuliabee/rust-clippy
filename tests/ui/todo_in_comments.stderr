error: TODO found in comment
  --> tests/ui/todo_in_comments.rs:1:1
   |
LL | // todo 0
   | ^^^^^^^^^
   |
   = note: `-D clippy::todo-in-comments` implied by `-D warnings`
   = help: to override `-D warnings` add `#[allow(clippy::todo_in_comments)]`

error: TODO found in comment
  --> tests/ui/todo_in_comments.rs:4:1
   |
LL | // TODO: 1
   | ^^^^^^^^^^

error: TODO found in comment
  --> tests/ui/todo_in_comments.rs:8:1
   |
LL | / /*
LL | | TODO: 2
   | | ^^^^
LL | |  */
   | |___^

error: TODO found in comment
  --> tests/ui/todo_in_comments.rs:12:5
   |
LL |     // TODO: 3
   |     ^^^^^^^^^^

error: TODO found in comment
  --> tests/ui/todo_in_comments.rs:13:5
   |
LL | /     /*
LL | |     TODO: 4
   | |     ^^^^
LL | |      */
   | |_______^

error: TODO found in comment
  --> tests/ui/todo_in_comments.rs:18:1
   |
LL | // TODO: 5
   | ^^^^^^^^^^

error: TODO found in comment
  --> tests/ui/todo_in_comments.rs:20:1
   |
LL | / /*
LL | | TODO: multi
   | | ^^^^
LL | | line
LL | | comment
LL | | containing
LL | | todo
   | | ^^^^
LL | |  */
   | |___^

error: TODO found in comment
  --> tests/ui/todo_in_comments.rs:28:1
   |
LL | / /*
LL | |  * TODO
   | |    ^^^^
LL | |  */
   | |___^

error: aborting due to 8 previous errors

